space-plugins/nuxt-base/server/utils/auth.ts

```diff
@@ -1,6 +1,5 @@
 import type { AuthHandlerParams } from '@storyblok/app-extension-auth';
 import type { AppConfigInput } from 'nuxt/schema';
-import { env } from './env';
```

Comment:
is this auto-imported?

Reply:
yes! i've added auto-import config for the server/utils